win-wpf/build.bat

```diff
@@ -3,6 +3,9 @@
 WHERE msbuild.exe > nul 2>&1
 IF %ERRORLEVEL% NEQ 0 set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin
 
+WHERE msbuild.exe > nul 2>&1
```

Comment:
Do we want to remove the above where block? They seem to be for the same condition

Reply:
one if the Community which looks like is the one used in github actions and the other one is for Enterprise which we use internally